instagram/src/components/PostContainer/PostContainer.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import './PostContainer.css'
+import { library } from '@fortawesome/fontawesome-svg-core'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import heart from '../SearchBar/heart.png'
+import speech from './speech.png'
+import { faIgloo } from '@fortawesome/free-solid-svg-icons'
+import CommentSection from '../CommentSection/CommentSection';
+import moment from 'moment';
+library.add(faIgloo);
+
+
+
+class PostContainer extends React.Component {
```

Comment:
If this is a container component, it should be purely presentational, meaning there will be no need for state. Since there are no lifecycle methods, this too can be a functional component